coherent_rates/isf.py

```diff
@@ -37,6 +48,30 @@
 _AX0Inv = TypeVar("_AX0Inv", bound=EvenlySpacedTimeBasis[Any, Any, Any])
 
 
+@dataclass
+class GaussianFitData:
+    """Represents the parameters from a Gaussian fit."""
+
+    amplitude: float
+    amp_error: float
+    sd: float
+    sd_error: float
+
+
+@dataclass
+class DoubleGaussianFitData:
```

Comment:
use tuple[GaussianFitData, GaussianFitData], and define closer to the first time it is used